tests/test_more.py

```diff
@@ -3218,8 +3218,8 @@ def test_math(self):
 class IsliceExtendedTests(TestCase):
     def test_all(self):
         iterable = ['0', '1', '2', '3', '4', '5']
-        indexes = list(range(-4, len(iterable) + 4)) + [None]
-        steps = [1, 2, 3, 4, -1, -2, -3, 4]
+        indexes = [-10, -6, -5, -4, -2, -1, 0, 1, 2, 4, 5, 6, 10, None]
```

Comment:
Why are we not testing between `-3` and `3` now?

Reply:
It seems to me that `3` and `-3` are unremarkable. We are testing `2` and `-2`, also the usual values. `0` is the edge case, `-1` and `1` are close to `0`. `5` and `-5` are the edge cases. `4`, `-4`, `5` and `-5` are close to these cases. `10` and `-10` are far beyond the borders.

Reply:
I'll approve and merge this if we put those back; I meant for that whole range to be tested. Missing `-4` was definitely a mistake, though!